packages/shared/config/package.json

```diff
@@ -8,8 +8,8 @@
   "main": "out/index.js",
   "type": "out/index.d.ts",
   "scripts": {
-    "postinstall": "tsc -p . --outDir out",
     "start": "tsc -p . --outDir out --watch",
+    "prebuild": "rm -fr out/*",
```

Comment:
Needed to force retranspilation (without it, type changes are not compiled if no code has been changed)